distances.py

```diff
@@ -0,0 +1,193 @@
+import numpy as np
+
+
+def values(hist_dataset, hist_im_query):
```

Comment:
rename to broadcast_query_values or remove and replace with `hist_im_query = hist_im_query[..., None]` inside every function